tests/test_resolve_names.cpp

```diff
@@ -40,9 +40,13 @@ TEST_F(ResolveNamesTest, UserFound)
         service().resolve_names("name", ews::search_scope::active_directory);
     auto resolution_mailbox = response.resolutions[0].mailbox;
     auto resolution_id = response.resolutions[0].directory_id;
+    auto resolution_contact = response.resolutions[0].contact;
     EXPECT_EQ(resolution_mailbox.name(), "User2");
     EXPECT_EQ(resolution_mailbox.value(), "User2@example.com");
     EXPECT_EQ(resolution_id.get_id(), "<GUID=abc-123-foo-bar>");
```

Comment:
This tests fails too. The node `t:Directory` is read instead of `t:DirectoryId` in `resolve_names_response_message::parse()` .